packages/npm-dependencies-validation/test/index.spec.ts

```diff
@@ -0,0 +1,98 @@
+import { getDependencyIssues } from "../src/index";
+import { expect, assert } from "chai";
+import * as fsPromises from "fs/promises";
+import { SinonMock, createSandbox } from "sinon";
+
+describe("index unit test", () => {
+  it("2 dependencies and 1 devDependency declared but are not installed", async () => {
+    const result = await getDependencyIssues({
+      fsPath: "./test/projects/no_deps_installed/package.json",
```

Comment:
evaluate some manner to link project (samples) files with test file.
otherwise overtime redundant samples will get collected

Reply:
Not sure what do you mean.
The test has relevant description and path to the tested folder

Reply:
i will try to implement this